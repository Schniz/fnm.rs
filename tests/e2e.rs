#[macro_use]
mod shellcode;

mod feature_tests;
