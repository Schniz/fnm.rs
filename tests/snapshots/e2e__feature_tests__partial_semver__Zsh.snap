---
source: tests/e2e.rs
expression: "&source.trim()"
---
set -e
eval "$(fnm env)"
fnm install 6
fnm use 6
if [ "$(node -v)" != "v6.17.1" ]; then
    echo 'Expected Node version to be "v6.17.1", Got: '"$(node -v)"
    exit 1
fi
